src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormatTest.java

```diff
@@ -0,0 +1,140 @@
+package io.cdap.plugin.gcp.bigquery.sink;
```

Comment:
Please include license  with year 2021. similar to 
https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java#L2

Reply:
added